lib/nimble_template/addons/variants/phoenix/mix_release.ex

```diff
@@ -27,33 +27,6 @@ defmodule NimbleTemplate.Addons.Phoenix.MixRelease do
   end
 
   defp edit_files(%{otp_app: otp_app, web_module: web_module} = project) do
-    Generator.delete_content(
-      "config/runtime.exs",
-      """
-      # Start the phoenix server if environment is set and running in a release
```

Comment:
should we keep deleting these content on Phoenix 1.6.11 @byhbt the purpose of this is delete the default config from Phoenix after generate owr project 🤔

Reply:
Yup, good catch, i revert the delete content here: https://github.com/nimblehq/elixir-templates/pull/250/commits/ea1c5ea6fc4df5c9561fa9badba7bb4337f3881b

However on the new version, the config content is changed. You can check here: https://www.phoenixdiff.org/?source=1.6.6&target=1.6.11 at `config/runtime.exs`

Reply:
Got it @byhbt

Reply:
Got it @byhbt